configure

```diff
@@ -608,10 +608,11 @@ if [ -z "${CC}" ] ; then
         exit 1
     fi
 else
-    ${CC} ${TMPCPROGC} ${OSCFLAGS} -o ${TMPCPROGB} 2>>/dev/null
+    CC_OUTPUT=$(${CC} ${TMPCPROGC} ${OSCFLAGS} -o ${TMPCPROGB})
```

Comment:
This change was to aide in understanding why some combinations of sanitizer flags were not working.

The change to this file can be omitted from the pull request now that I have things working. Let me know if you want this file reverted.

Reply:
Got it. I have no strong preference.